packages/docs/docs/compliance/onc.md

```diff
@@ -44,7 +44,7 @@ This is the list of certified criteria. Related to this criteria these scripts a
 | g5                                                  | Accessibility-Centered Design                        |
 | [g10](/docs/api)                                    | Standardized API for Patient and Population Services |
 
-The following criteria are required for CHPL listing: [d12](#encrypt-authentication-credentials-d12), [d13](#multi-factor-authentication-d13), [g4](#quality-management-system-g4), [g5](#accessibility-centered-design-g5).
```

Comment:
@reshmakh  - FYI - New version of Docusaurus has more sophisticated broken link detection.  I didn't see any obvious replacements, so I removed the links entirely.